commercelayer/commercelayer-react-components
# Move README.md under 'react-components' package folder

## What I did

I moved the README.md under the 'react-components' package folder so that the README will be visible at NPM. I also linked back the README to the root folder by using a symlink.

## How to test



After publishing the package with this change, you should be able to see the package's README.md when you land on the [NPM package page](https://www.npmjs.com/package/@commercelayer/react-components).

## Checklist
